_posts/2017-12-10-trl-mesh-test.md

```diff
@@ -10,4 +10,4 @@ startTime: '13:00'
 endTime: '16:00'
 ---
 
-Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} to help deploy and test the first at-scale Toronto Mesh mesh. Help us discover its capabilities, downfalls and maybe learn a thing or two you didn't know before!  Our end goal with these tests is to learn enough to deploy a Toronto Mesh at 2018's Maker Festival which happens in the Toronto Reference Library in the summer. Meet us at [{{ page.location }}]({{ page.locationLink }}) and help out. Don't forget to bring your nodes, laptops, phones and other Internet Capable devices!
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} to help deploy and test the first at-scale Toronto Mesh network. Help us discover its capabilities, downfalls and maybe learn a thing or two you didn't know before!  Our end goal with these tests is to learn enough to propose a Toronto Mesh deployment at 2018's Maker Festival which happens in the Toronto Reference Library in the summer. Meet us at [{{ page.location }}]({{ page.locationLink }}) and help out. Don't forget to bring your nodes, laptops, phones and other Internet capable devices!
```

Comment:
Extra space `before!  Our`

Reply:
`s/happens in the/happens at the`

Reply:
Yeh I caught the happens in/at one.  Still in the habit of double spacing after punctuations :+1: